main.go

```diff
@@ -45,6 +47,12 @@ const (
 	urlChangeTariff = "https://my.yota.ru/selfcare/devices/changeOffer"
 )
 
+func (tariff Tariff) String() string {
```

Comment:
What is the point of this implementation? I think fmt.Sprintf("#+v", tariff) does the same, no?

Reply:
just for test
tests is not committed now

Reply:
you're right. added because  i have a large java-background :disappointed: